tests/specs/utils/http.spec.js

```diff
@@ -114,18 +114,22 @@ describe('HTTP', () => {
             });
         });
 
+        function randomIntBetween(min, max) {
+            return Math.floor(Math.random() * (max - min)) + min;
+        }
+
         describe('#handleStatus', () => {
-            for (let status = 200; status < 300; status++) {
+            [200, randomIntBetween(201, 299), 299].forEach((status) => {
```

Comment:
Disclaimer: I'm just commenting to play devil's advocate, I don't really care if you do it this way.

> When you get down to it, the test is ensuring our correct use of `>=` and `<` operators and their operands, and defending against regressions that change these bounds.

This assumes that the underlying implementation is actually using those operators, or some form of range builder. A unit test shouldn't make assumptions about the underlying implementation. For example, the code could have been implemented as `if(statusCode === 200 || statusCode === 299)` and your test would pass even though `201` - `298` are not correctly handled